slash/log.py

```diff
@@ -106,6 +106,11 @@ def emit(self, record):
             reporter.notify_after_console_output()
         return returned
 
+def _slash_logs_filter(record, handler): # pylint: disable=unused-argument
+    return record.extra.get('filter_bypass') or \
+           record.level >= config.root.log.filter_log_level or \
```

Comment:
the `or` ordering should be reversed IMO. set lookup for channel name is much faster than the mechanism implementing `__getattr__` for `config.root`